### DISCUSSION TITLE
Can't install CLI for Zed after installing via pacman

### DISCUSSION THREAD

**Discussion by thisjustinf** [zed-industries/zed#14357] [Feature Requests]
It's pretty straightforward but the error message I get when I try to install from the control pane is:

![image](https://github.com/user-attachments/assets/d134b486-fefb-44e0-b5bb-905508286282)

Is there a way to solve this? I very much prefer to keep all software and applications managed through a package manager if possible

**Comment by thisjustinf** [comment]
Never mind, in arch linux the CLI command is zeditor after installing via pacman so I just use `alias zed="zeditor"`

**Comment by Testare** [comment]
Now the problem I have is that "zeditor" is the gui one and I can't find the cli version >,<
